src/components/common/landingPage/trending/Trending.js

```diff
@@ -17,7 +17,7 @@ const TrendingArticles = (props) => {
       await props.getTrendingArticles();
     };
     fetchTrendingArticles();
-  }, []);
+  }, [props]);
```

Comment:
Why did you include `props` here?

Reply:
That's Pelumi's code, I guess he wants the component to call the function whenever the props change

Reply:
I've fixed this